src/Microsoft.Diagnostics.TestHelpers/TestHost/TestHost.cs

```diff
@@ -42,19 +44,19 @@ public ITarget Target
             }
         }
 
-        public bool IsTestDbgEng => Config.AllSettings.TryGetValue("TestDbgEng", out string value) && value == "true";
+        public abstract ImmutableArray<string> ExecuteHostCommand(string commandLine);
 
         protected abstract ITarget GetTarget();
 
         public string DumpFile => TestConfiguration.MakeCanonicalPath(Config.AllSettings["DumpFile"]);
 
-        public string TestDataFile => TestConfiguration.MakeCanonicalPath(Config.AllSettings["TestDataFile"]);
+        public string TestDataFile => TestConfiguration.MakeCanonicalPath(Config.AllSettings.GetValueOrDefault("TestDataFile"));
 
         public override string ToString() => DumpFile;
     }
 
     public static class TestHostExtensions
     {
-        public static bool IsTestDbgEng(this TestConfiguration config) => config.AllSettings.TryGetValue("TestDbgEng", out string value) && value == "true";
+        public static bool IsTestDbgEng(this TestConfiguration config) => config.AllSettings.GetValueOrDefault("TestDbgEng", string.Empty) == "true";
```

Comment:
The TestConfiguration properties are usually compared with the string "true" in the rest of the code.  I'm not sure any more is necessary and it can be blank/empty. I think Convert.ToBoolean makes all of that more complex.